src/GafferArnold/IECoreArnoldPreview/Renderer.cpp

```diff
@@ -527,6 +527,17 @@ IECore::InternedString g_curvesModeAttributeName( "ai:curves:mode" );
 IECore::InternedString g_linkedLights( "linkedLights" );
 IECore::InternedString g_lightFilterPrefix( "ai:lightFilter:" );
 
+const AtString g_nodeArnoldString("node");
+const AtString g_ginstanceArnoldString("ginstance");
+const AtString g_polymeshArnoldString("polymesh");
+const AtString g_curvesArnoldString("curves");
+const AtString g_boxArnoldString("box");
+const AtString g_volumeArnoldString("volume");
+const AtString g_sphereArnoldString("sphere");
+const AtString g_subdiv_typeArnoldString("subdiv_type");
```

Comment:
OK - I felt like these might be an exception because the meaning of this name is literally 'a constant which stores the letters "subdiv_type" as an Arnold string', but I can switch them if it bugs you.